docs/sources/configure-client/grafana-alloy/_index.md

```diff
@@ -81,6 +81,18 @@ In pull mode, the collector periodically retrieves profiles from Golang applicat
 1. Ensure Alloy is set to pull mode and targeting the correct pprof endpoints. For step-by-step instructions, visit the [Go (Pull Mode)](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/grafana-alloy/go_pull) documentation.
 1. The collector queries the pprof endpoints of your Golang application, collects the profiles, and forwards them to the Pyroscope server.
 
+## Receiving profiles from Pyroscope SDKs
```

Comment:
We've moved away from using -ing verbs in headings. We use imperative forms: Receive instead of Receiving. For more information, https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/
```suggestion
## Receive profiles from Pyroscope SDKs
```